tests/plugins/test_plugins_manager.py

```diff
@@ -30,24 +30,14 @@
 from tests.test_utils.config import conf_vars
 from tests.test_utils.mock_plugins import mock_plugin_manager
 
-importlib_metadata_string = 'importlib_metadata'
-
-try:
+if sys.version_info[:2] >= (3, 8):
```

Comment:
I think we have PY38 constant for that in `airflow':  @khalidmammadov . 

Also @ashb - I recall discussions we had about it, I think it makes sense to based it on  Python version

Reply:
Yeah, someone (i forget who, maybe Martjin Peiters? sp?) recommended the approach TP says -- always try the third part first, as that API is stable and predictable.)